### ISSUE TITLE
MSFVENOM - linux/armle/cmd payload PrependSetuid option does not result in different output when included in the command

### ISSUE THREAD

**Issue by nstarke** [rapid7/metasploit-framework#12777]
Try running these two commands:

```
msfvenom -p linux/armle/exec CMD=/bin/sh PrependSetuid=True -f elf | shasum
msfvenom -p linux/armle/exec CMD=/bin/sh -f elf | shasum
```

The output will hash to the same value, which I would not expect.  I would expect the first command to have additional bytes for the `Setuid` code.  

Notice that when these commands are run, the hash output is different:

```
msfvenom -p linux/x86/exec CMD=/bin/sh PrependSetuid=True -f elf | shasum
msfvenom -p linux/x86/exec CMD=/bin/sh -f elf | shasum
```

I'm guessing whats needed is support for this particular encoder for the PrependSetuid option, but I'm not certain.

**Comment by timwr** [comment]
This isn't implemented on armle yet but it should be straightforward to add it:
https://github.com/rapid7/metasploit-framework/blob/master/lib/msf/core/payload/linux.rb#L432

**Comment by nstarke** [comment]
Hello,  I wanted to reach out and see what needs to be done on this since it is assigned to me. I'm not quite sure how to proceed from here.  Thanks!

**Comment by bcoles** [comment]
> 
> 
> Hello, I wanted to reach out and see what needs to be done on this since it is assigned to me. I'm not quite sure how to proceed from here. Thanks!

Is this issue resolved by #12779 ? If so, you've already done what needs to be done, and it is up to someone from the team to review.

Your code looks fine, but I don't speak armle, and haven't tested it.
